File: day1/ruby/solution.rb
file = File.open('input.txt')
file_data = file.readlines.map(&:chomp)
increased = 0
(1..file_data.length).each do |i|
  increased += 1 if file_data[i].to_i > file_data[i - 1].to_i
end
puts increased

File: day1/solution.js
const { readFileSync } = require("node:fs");
const input = readFileSync("./input.txt", "utf-8").split("\n");
let increased = 0;
for (let i = 1; i < input.length; i++) {
  if (input[i] && input[i - 1] && (+input[i] > +input[i - 1])) increased++;
}
console.log(increased);
